modules/distribution/resources/operation_policies/specifications/ccCallInterceptorService.json

```diff
@@ -15,7 +15,7 @@
     {
       "name": "includes",
       "displayName": "Includes to Payload",
-      "description": "Includes to the request payload of interceptor service as comma separated ",
+      "description": "Includes a comma separated list to the request payload of interceptor service",
```

Comment:
The idea of this should be something,
`A comma separated list of items (for example request_header, request_body, response_header, response_body) to the request payload of interceptor service`


So this is an example
```yaml
-
  policyName: ccCallInterceptorService
  policyId: 97dc8a81-5a69-4164-8715-211ef67169de
  order: 7
  parameters:
    interceptorServiceURL: https://hello-interceptor:8080
    includes: request_headers,request_body
```

Reply:
Updated with https://github.com/wso2/product-apim/pull/12646/commits/c9db8c878b4cca25ba6abe7a50c232715a39c465